src/miqclient/api.py

```diff
@@ -21,7 +21,7 @@ class APIException(Exception):
 
 
 class API(object):
-    def __init__(self, entry_point, auth, logger=None):
+    def __init__(self, entry_point, auth, logger=None, verify=True, certfile_path=None):
```

Comment:
@cben thanks for the review.
I've made the changes you proposed, with a small difference of s/verify/verify_ssl